general/clients/clients.md

```diff
@@ -159,10 +159,11 @@ The official Jellyfin Kodi plugin.
 4. Within a few seconds you should be prompted for your server-details.
     * If a Jellyfin server is detected on your local network, it will displayed in the popup
     * If a Jellyfin server is not detected on your local network, select "Manually add server"
+        * Note that `/jellyfin` in these examples should match the `baseurl` field in the server (defaults to `/jellyfin`)
         * Enter the server name or IP address and the port number (default is 8096)
+            * Host: `192.168.1.10:8096/jellyfin`
         * If using SSL and a reverse proxy, enter the full URL scheme in the "Host" field
-            * Host: https://jellyfin.example.com
-            * Port: 443
+            * Host: `https://jellyfin.example.com/jellyfin`
```

Comment:
Since we have changed the default back to `/` I don't think we should leave `/jellyfin` in the documentation here.

Reply:
Ah, didn't realized we'd changed the default back.  I'll change this part